src/helpers/util.js

```diff
@@ -293,12 +293,13 @@ class Util {
 
   /**
    * @param {Object|Array} config
-   * @param projectName
+   * @param {String}projectName
+   * @param {Number} listLimit
    */
-  static printListAuto(config, projectName) {
+  static printListAuto(config, projectName, listLimit) {
```

Comment:
add default value 5 for listLimit parameter
```suggestion
  static printListAuto(config, projectName, listLimit = 5) {
```